pkg/forklift-api/webhooks/mutating-webhook/mutators/ovirt-cert-mutator_test.go

```diff
@@ -0,0 +1,34 @@
+package mutators
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/onsi/gomega"
+)
+
+func TestCertAppanding(t *testing.T) {
+	g := gomega.NewGomegaWithT(t)
+
+	providedCa, err := ioutil.ReadFile("originalCert.pem")
+	if err != nil {
+		panic(err)
```

Comment:
if we use gomega (which is cool!) we can check that no error have-occurred

Reply:
cool, wasn't aware of it, changing.